.gitignore

```diff
@@ -1,10 +1,6 @@
-.idea
-.DS_Store
-build
-phpunit.xml
-Resources/doc/_build/*
-nbproject
-coverage
+/build
+/Resources/doc/_build
+/vendor
 composer.lock
-vendor
+phpunit.xml
 .php_cs.cache
```

Comment:
Please sort the files as well like in https://github.com/sonata-project/SonataUserBundle/blob/3.x/.gitignore